be/src/exprs/agg/java_udaf_function.cpp

```diff
@@ -125,6 +125,46 @@ template jvalue cast_to_jvalue<true>(MethodTypeDescriptor method_type_desc, cons
 
 template jvalue cast_to_jvalue<false>(MethodTypeDescriptor method_type_desc, const Column* col, int row_num);
 
+void assign_jvalue(MethodTypeDescriptor method_type_desc, Column* col, int row_num, jvalue val) {
+    auto& helper = JVMFunctionHelper::getInstance();
+    Column* data_col = col;
+    if (col->is_nullable() && method_type_desc.type != PrimitiveType::TYPE_VARCHAR &&
+        method_type_desc.type != PrimitiveType::TYPE_CHAR) {
+        auto* nullable_column = down_cast<NullableColumn*>(col);
+        nullable_column->set_null(row_num);
```

Comment:
is cell located at row num always null?  
```
if (val.l == nullptr) {
       nullable_column->set_null(row_num);
}
```

Reply:
> is cell located at row num always null?
yes it always true, There is no pre-allocated memory space for TYPE_CHAR or TYPE_VARCHAR.
so if we call nullable_column->set_null(row_num) for TYPE_CHAR or TYPE_VARCHAR, which will got a wrong result
https://github.com/StarRocks/starrocks/blob/main/be/src/exec/vectorized/analytor.cpp#L401